.github/workflows/check-and-lint.yaml

```diff
@@ -171,4 +171,4 @@ jobs:
         run: cargo run -- --manifest-path sample_rlib/Cargo.toml --att
 
       - name: cdylib project
-        run: cargo run -- --manifest-path sample_cdylib/Cargo.toml --everything
```

Comment:
If there's more than one name in a library either `--everything` or a name is needed, to the test to succeed. I think it makes sense to add tests for `add` and `_mul`. Similar problem with the Linux test.

Reply:
Just added test for `add` & `_mul` for linux & macos.

Reply:
Yup, looks good!